src/NetscriptFunctions/Sleeve.ts

```diff
@@ -174,6 +172,84 @@ export function NetscriptSleeve(player: IPlayer, workerScript: WorkerScript, hel
 
       return player.sleeves[sleeveNumber].workoutAtGym(player, gymName, stat);
     },
+    setToWork: function (_sleeveNumber: number, _sleeveTask: SleeveTask): boolean {
```

Comment:
Should be named `setToTask`, as the functionality covers more than just working for companies/factions.

But also, I'm not sure if this is even an appropriate addition as it's redundant with all the individual `sleeve.setTo*` functions. The requested feature in #3566 is something a player can implement themselves on top of the existing Sleeve API, and in my opinion should have to if they want to have a helper function that works in that specific way.

Reply:
That's a sound argument. Phyzical had a similar reaction. And in fact, I think so too. (more than  half the reason I did that coding in the first place was to familiarize myself with netscript)

I will revert these change. #3566 will be discarded. (well, at least the underlying issue should be fixed by patching the SleeveTask interface)